src/client/Microsoft.Identity.Client/ApiConfig/AbstractPublicClientAcquireTokenParameterBuilder.cs

```diff
@@ -21,6 +23,23 @@ internal AbstractPublicClientAcquireTokenParameterBuilder(IPublicClientApplicati
             PublicClientApplicationExecutor = publicClientApplicationExecutor;
         }
 
+        public AbstractPublicClientAcquireTokenParameterBuilder<T> WithPoPAuthenticationScheme(Uri uri) // TODO: naming and comments
+        {
+            CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithScheme);
+            CommonParameters.AuthenticationScheme = new PoPAuthenticationScheme(uri, null); // TODO add a default crypto provider
+
+            return this;
+        }
+
+        // Allows testing the PoP flow with any crypto. Consider making this public.
+        internal AbstractPublicClientAcquireTokenParameterBuilder<T> WithPoPAuthenticationScheme(Uri protectedUri, IPoPCryptoProvider popCryptoProvider) 
```

Comment:
**Proposal for XML comment**
summary: Modifies the token acquisition request so that the acquired token is a Proof of Possession token (PoP), rather than a bearer token. The keys used in signing the Pop token are provided by popCryptoProvider. See https://aka.ms/msal-net-pop
uri: URI for which to request a Proof of Possession token is aquired
popCryptoProvider: key provider used to sign the Pop token.
See also: AuthorizationResult.CreateAuthorizationHeader